app/views/layouts/application.html.erb

```diff
@@ -53,12 +53,16 @@
         </form> -->
         <ul class="nav navbar-nav navbar-right">
           <% if current_user %>
+            <% unless current_user.subscribed %>
+              <li><a href="/account">Subscribe to Pro</a></li>
+              <% else %>
+                <li><%= link_to current_user.email, account_path %></li>
```

Comment:
From your email: 1) So what I currently call the “sales page” is what anonymous users and unsubscribed users see when viewing the account page.

Are you asking for a separate account page for "unsubscribed but signed in" users instead?